webhooks.md

```diff
@@ -0,0 +1,91 @@
+# Webhook integration
+
+## Endpoints
+
+Events will be pushed to the endpoint URL defined in organization settings in the workshop tool. Preferably the URL contains some non-guessable string unique to this integration.
+
+Request Headers:
+* `X-Request-Id`: Identifier for this request
+
+Acceptable return HTTP codes:
+* 200: Messages have been consumed successfully
+* not 200: Something went wrong, retry
+
+Requests will be retried 3 times after 20 seconds delay. If no response is received on the first 3 retry attempts, each following attempt is made afterwards with an exponential back off starting at 2 minutes.
+
+Messages will be one at a time. Order is not guaranteed, but timestamps are included in the message.
+
+Requests should only be rejected in case you have issues parsing, validating or storing the event, for example not recognizing the VIN should not result in rejection as this is a configuration synchronization issue and not related to the transfer of the messages.
```

Comment:
```suggestion
Requests should only be rejected in case you have issues parsing, validating or storing the event, for example not recognizing the VIN should not result in a rejection as this is a configuration synchronization issue and not related to the transfer of the messages.
```